app/assets/stylesheets/_maps.css.scss

```diff
@@ -0,0 +1,35 @@
+#map-canvas {
+  height: 300px;
+  width: 100%;
+}
+
+.controls {
+  -moz-box-sizing: border-box;
```

Comment:
Properties should be ordered border, border-radius, box-shadow, -moz-box-sizing, box-sizing, height, margin-top, outline<br>Avoid vendor prefixes.